src/Uno.UI/UI/Xaml/IFrameworkElementImplementation.iOS.tt

```diff
@@ -450,7 +450,8 @@ namespace <#= mixin.NamespaceName #>
 					SizeChanged?.Invoke(this, new SizeChangedEventArgs(previousSize, _actualSize));
 
 #if <#= mixin.IsFrameworkElement #>
-					if (Superview != null && !(Superview is DependencyObject))
+					// Require measurement from CommandBar elements just cause apps to crash
```

Comment:
Does the comment mean that we should never remeasure UINavigationBar ?  (I figured it out because of the diff but I'm not sure it's clear. Do we know why?

Reply:
Apart from being a monstrous hack, this will cause a regression when the CommandBar content is text that's too long to display in portrait.

Reply:
I've updated my PR. Please see new modifications